torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -1579,8 +1583,8 @@ def _profiler_test_with_rpc(self, rpc_exec_mode, func, args, use_record_function
                 scope_event = get_function_event(events, "foo")
                 # Since RPC call is within the scope, its CPU interval should be
                 # contained within foo's interval.
-                self.assertTrue(scope_event.time_range.start < rpc_event.time_range.start)
-                self.assertTrue(scope_event.time_range.end > rpc_event.time_range.end)
+                self.assertLessEqual(scope_event.time_range.start, rpc_event.time_range.start)
+                self.assertGreaterEqual(scope_event.time_range.end, rpc_event.time_range.end)
```

Comment:
curious, did real tests hit equality here?

Reply:
Unfortunately, the stack trace only gives us:

```
Dec 18 23:45:14   File "/opt/conda/lib/python3.6/site-packages/torch/testing/_internal/distributed/rpc/rpc_test.py", line 1583, in _profiler_test_with_rpc
Dec 18 23:45:14     self.assertTrue(scope_event.time_range.end > rpc_event.time_range.end)
Dec 18 23:45:14   File "/opt/conda/lib/python3.6/unittest/case.py", line 682, in assertTrue
Dec 18 23:45:14     raise self.failureException(msg)
Dec 18 23:45:14 AssertionError: False is not true
Dec 18 23:45:14 exiting process with exit code: 10
```

and the error has not reproduced since then. Though I remember seeing some other profiling tests hit equality, possibly due to the actual numerical difference being very small.